src/Sarif.FunctionalTests/UpdateBaselines.ps1

```diff
@@ -54,17 +61,16 @@ function Build-Baselines($toolName)
         # Actually run the converter
         Remove-Item $outputTemp -ErrorAction SilentlyContinue
         &$utility convert "$input" --tool $toolName --output "$outputTemp" --pretty-print
-
         # Next, perform some rewriting. The PREfast converter in particular cannot embed file contents as the source      
         # SARIF emitted by the compiler does not contain the optional 'files' member of the 'run' object.
-        &$utility rewrite ""$outputTemp"" --output ""$outputTemp"" --pretty-print --insert "TextFiles;Hashes;RegionSnippets;ContextCodeSnippets" --force
+        &$utility rewrite ""$outputTemp"" --output ""$outputTemp"" --pretty-print --insert """TextFiles;Hashes;RegionSnippets;ContextRegionSnippets""" --force
```

Comment:
>""" [](start = 91, length = 3)

If the reason for this change is that Powershell interpreted the `;` as a statement terminator, you could instead escape the `;` with a backtick: ``--insert TextFiles`;Hashes`;RegionSnippets`;ContextRegionSnippets``